Formula/rocksdb.rb

```diff
@@ -69,16 +69,17 @@ def install
                                 "-std=c++11", "-stdlib=libc++", "-lstdc++",
                                 "-lz", "-lbz2",
                                 "-L#{lib}", "-lrocksdb_lite",
+                                "-DROCKSDB_LITE=1",
```

Comment:
This fixes the test failures in earlier attempts to upgrade (e.g. #56602).

Reply:
Nice, good to know.